### PR TITLE
#167190432 Recieve verification notification

### PR DESCRIPTION
**Pull request by cvjude**
#### What does this PR do?
- Should allow a user to verify his email when registered for the App

#### How can this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Run the command below to test
```
npm test
```

#### Any background context you want to provide?
none

#### What are the relevant Pivotal Tracker stories? 
- [#167190432](https://www.pivotaltracker.com/story/show/167190432)

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -2,3 +2,4 @@ PORT = an integer representing the port through which the server will be accesse
 DATABASE_URL = postgres://user_name:password@host:port/db_name
 TEST_DATABASE_URL = postgres://user_name:password@host:port/test_db_name
 SECRET_KEY = a string representing the secret key for decoding JWTs on the server
+APP_URL=the app url
```
Comment (topseySuave) [inline .env.sample]: Try to apply a new lone at the end of every file.

File: mail.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <div class=button>
+        <a class=link href="google.com">Confirm Email Adress</a>
+      </div>
+    </div>
+  </div>
+
+
+</body>
+
+</html>
```
Comment (topseySuave) [inline mail.html]: Here also.

Comment (cvjude) [reply mail.html]: Owk it will get on it now

File: server/routes/user.js [hunk 3]
```diff
@@ -9,8 +9,7 @@ const { signUp, login, logout } = UserController;
 
 userRoute.post('/', validateUser, signUp);
 userRoute.post('/login', validateLogin, login);
-
-// logs out a user
+userRoute.get('/confirmEmail', UserController.confirmEmail);
```
Comment (chuxmykel) [inline server/routes/user.js]: Hey @cvjude. Great job with the implementation. It's really awesome. I love the mail template. Great Job. I think you missed the `UserController.confirmEmail` line. I suggest you add it to the list of destructured items and call it accordingly.

**Review by chuxmykel — APPROVED** [review]
Just that little change and it's good to go. Great job

File: server/routes/user.js (hunk 3 above)
Comment (cvjude) [reply server/routes/user.js]: Thanks, how did I forget that. I'll get on it as soon as I can

**Review by Hadeneekeh — APPROVED** [review]

**Review by encodedBicoding — APPROVED** [review]

**Review by Lundii — APPROVED** [review]
Working fine when tested on my Local. Good job

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by chiboycalix — APPROVED** [review]
LGTM

**Review by topseySuave — APPROVED** [review]

**Pull request merged**